.github/workflows/uc_loadsim.yml

```diff
@@ -30,9 +37,528 @@ defaults:
     shell: bash
 
 jobs:
-  kubernetes-psql:
+  docker-compose-sqlite:
     runs-on: ubuntu-latest
     steps:
+    - name: Set EC2 facts
+      id: ec2_facts
+      run: |
+        AMI=$(aws ec2 describe-images --filters '[{"Name":"description","Values":["Canonical, Ubuntu, 20.04 LTS, amd64*"]},{"Name":"owner-id","Values":["099720109477"]}]' --query 'sort_by(Images, &CreationDate)[-1].ImageId' --output text)
+        INSTANCE_NAME=owgw-dc-sqlite-${{ github.run_number }}
+        echo "::set-output name=ami::${AMI}"
+        echo "::set-output name=instance_name::${INSTANCE_NAME}"
+
+    # Create EC2 infrastructure
```

Comment:
Think it would make sense to put this in a composite action at some point since this could be re-used in other workflows as well.

Reply:
Yes, probably it makes sense to do so